examples/spacy/text_classification.py

```diff
@@ -0,0 +1,135 @@
+    if model is not None:
+        # Load existing spaCy model
+        nlp = spacy.load(model)
+        print("Loaded model '%s'" % model)
+    else:
+        # Create blank Language class
+        nlp = spacy.blank("en")
+        print("Created blank 'en' model")
+
+    # Add the text classifier to the pipeline if it doesn't exist
+    # nlp.create_pipe works for built-ins that are registered with spaCy
+    if "textcat" not in nlp.pipe_names:
+        textcat = nlp.create_pipe("textcat")
+        nlp.add_pipe(textcat, last=True)
+    # Otherwise, get it, so we can add labels to it
+    else:
+        textcat = nlp.get_pipe("textcat")
+
+    # Add label to text classifier
+    textcat.add_label("POSITIVE")
+
+    # Load the IMDB dataset
+    print("Loading IMDB data...")
+    (train_texts, train_cats), (dev_texts, dev_cats) = load_data(limit=n_texts)
+    print("Using {} examples ({} training, {} evaluation)"
+          .format(n_texts, len(train_texts), len(dev_texts)))
+    train_data = list(zip(train_texts,
+                          [{"cats": cats} for cats in train_cats]))
+
+    # Get names of other pipes to disable them during training
+    other_pipes = [pipe for pipe in nlp.pipe_names if pipe != 'textcat']
+    with nlp.disable_pipes(*other_pipes):  # only train textcat
+        with mlflow.start_run():
+            optimizer = nlp.begin_training()
+            print("Training the model...")
+            print("{:^5}\t{:^5}\t{:^5}\t{:^5}".format("LOSS", "P", "R", "F"))
+            mlflow.log_param("n_texts", n_texts)
+            mlflow.log_param("n_iter", n_iter)
+            for i in range(n_iter):
+                losses = {}
+                # Batch up the examples using spaCy's minibatch
+                batches = minibatch(train_data, size=compounding(4., 32., 1.001))
+                for batch in batches:
+                    texts, annotations = zip(*batch)
+                    nlp.update(texts, annotations, sgd=optimizer, drop=0.2,
+                               losses=losses)
+                with textcat.model.use_params(optimizer.averages):
+                    # Evaluate on the dev data split off in load_data()
+                    scores = evaluate(nlp.tokenizer, textcat, dev_texts, dev_cats)
+                # Print a simple table
+                print("{0:.3f}\t{1:.3f}\t{2:.3f}\t{3:.3f}"
+                      .format(losses["textcat"], scores["textcat_p"],
+                              scores["textcat_r"], scores["textcat_f"]))
+
+                mlflow.log_metric("loss", losses["textcat"])
+                mlflow.log_metric("precision", scores["textcat_p"])
+                mlflow.log_metric("recall", scores["textcat_r"])
+                mlflow.log_metric("f-score", scores["textcat_f"])
+
+            # Test the trained model
+            test_text = "This movie sucked"
+            doc = nlp(test_text)
+            print(test_text, doc.cats)
+
+            mlflow.spacy.log_model(nlp, "model")
```

Comment:
Please extend this example to load this saved model and predict.
alternatively, add a unit test to cover the log_model and load_model flow.